examples/platforms/cc2538/radio.c

```diff
@@ -262,6 +262,21 @@ void setTxPower(int8_t aTxPower)
     }
 }
 
+static bool cc2538SrcMatchEnabled(void)
+{
+    return (HWREG(RFCORE_XREG_FRMCTRL1) & RFCORE_XREG_FRMCTRL1_PENDING_OR) == 0;
+}
+
+static bool cc2538GetSrcMatchFoundIntFlag(void)
+{
+    bool flag = (HWREG(RFCORE_SFR_RFIRQF0) & RFCORE_SFR_RFIRQF0_SRC_MATCH_FOUND) != 0;
+    if (flag)
+    {
+        HWREG(RFCORE_SFR_RFIRQF0) &= ~RFCORE_SFR_RFIRQF0_SRC_MATCH_FOUND;
+    }
```

Comment:
is it necessary?

Reply:
yes. the function is supposed to read and clear the flag

Reply:
what's the impact if not?

would this flag would be refreshed per receive action by the PHY?

Reply:
I don't think so

Reply:
@librasungirl verify from test on CC2538 devices that if the flag is not cleared, it will be always 1 once it is set.